### ISSUE TITLE
takeWhile Operator - not supporting inclusive inconsole at rxjs site

### ISSUE THREAD

**Issue by avrim-tr** [ReactiveX/rxjs#6730]
### Describe the bug

rxjs.of(1,2,3,4).pipe(

rxjs.operators.takeWhile(val => val !== 3, true)
).subscribe(x => console.log(x))

Using in internal apps work as expeceted.

### Expected behavior

Should log:
1,2,3

### Version

V6 and V7.1.0

**Comment by martinsik** [comment]
Looks like rxjs.dev site is using RxJS 6.3.3:

![image](https://user-images.githubusercontent.com/238765/147463650-024da74e-ab72-47bf-98ff-728e3acf952e.png)

The optional `inclusive` option was added in [RxJS 6.4.0](https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#640-2019-01-30) according to the changelog.

**Comment by benlesh** [comment]
Yeah, we actually want to remove this feature entirely anyhow. It's fairly unnecessary. People should just use stackblitz.com/fork/rxjs

**Comment by pertrai1** [comment]
Looks like this was changed here: https://github.com/ReactiveX/rxjs/commit/e7f59fd0599b13de16f17a63b98a865fdee67806#diff-44ce9f855a5470017be4dae2d43113eaad1451772c6295f9246e969a100c160d if I am reading this correctly.

**Comment by benlesh** [comment]
Hi @avrim-tr — thank you for taking the time to open this issue and for sharing your thoughtful feedback with the RxJS project. I’m sorry it took us so long to respond. Given how much time has passed and the lack of recent activity, I’m going to close this for now so the tracker reflects the work we’re likely to act on. If this is still relevant on a current RxJS release, please feel welcome to open a new issue with updated context and a reproduction. Thank you again for your contribution and patience.

— Ben
